zerver/models/messages.py

```diff
@@ -244,7 +244,6 @@ class Meta:
                 name="zerver_message_realm_id",
             ),
             models.Index(
-                # Used by 0680_rename_general_chat_to_empty_string_topic
```

Comment:
I'm actually wondering if we should add a comment here to mention that this has helped in `0501` in the past (we didn't include it at that time) and we expect it to be helpful in future migrations / queries. 

ref: [#backend > general chat - migration discussion @ 💬](https://chat.zulip.org/#narrow/channel/3-backend/topic/general.20chat.20-.20migration.20discussion/near/2104032)

> Specifically, the migration added in [3db1733310ddd944c2e690ba673232345c928eec](https://github.com/zulip/zulip/commit/3db1733310ddd944c2e690ba673232345c928eec) needed that partial index to be able to complete in any reasonable amount of time. We did not roll that partial index into the migration, though we potentially should have.